Translation revisions (#559)
07th-mod/higurashi-console-arcs: Update/koto_004.txt

//r陸は困ったように頭をかいて、視線を泳がせている。
	OutputLine(NULL, "陸は困ったように頭をかいて、視線を泳がせている。",

> 			NULL, "Riku's eyes darted around as he scratched his head with a troubled look on his face.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Riku's eyes darted around as he scratched his head, with a troubled look on his face.", Line_Normal);
```

//r実は口から出まかせのひっかけだったのだが、…お人よしというか、嘘のつけない人だ。
	OutputLine(NULL, "実は口から出まかせのひっかけだったのだが、…お人よしというか、嘘のつけない人だ。",

> 			NULL, "Actually, I asked it in ignorance of that fact and bluffed him... but Riku was soft on others and a poor liar.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Actually, I had asked without knowing for sure and bluffed him... but Riku was soft on others and a poor liar.", Line_Normal);
```
